test/test_cursor.py

```diff
@@ -1408,6 +1416,14 @@ def test_command_cursor_to_list_empty(self):
         docs = c.to_list()
         self.assertEqual([], docs)
 
+    @client_context.require_change_streams
+    def test_command_cursor_to_list_length(self):
+        # Set maxAwaitTimeMS=1 to speed up the test.
+        c = self.db.test.aggregate([{"$changeStream": {}}], maxAwaitTimeMS=1)
+        self.addCleanup(c.close)
+        docs = c.to_list(2)
+        self.assertEqual(len(docs), 2)
```

Comment:
How long does this test take on a sharded cluster?

Reply:
It doesn't work on sharded, I'll make a separate test for CommandCursor